### ISSUE TITLE
Please provide api to send Authorization headers while sending POSTs to server. 

### ISSUE THREAD

**Issue by ronaksvyas** [dtschust/redux-bug-reporter#22]

**Comment by ronaksvyas** [comment]
Something like: 

`var createSubmit = function createSubmit(url, headers = {
	        'Accept': 'application/json',
	        'Content-Type': 'application/json',
	        'Authorization' : getAuthToken()
	      } ) {

	  return function (newBug) {
	    return fetch(url, {
	      method: 'post',
	      headers: headers,
	      body: JSON.stringify(newBug)
	    }).then(function (response) {
	      if (!response.ok) {
	        throw Error(response.statusText);
	      }
	      return response.json();
	    });
	  };
};`

**Comment by dtschust** [comment]
Hi @ronaksvyas, thanks for filing an issue! I'd like to make this solution a little more generic, how does this sound?

```
const createSubmit = ({url, fetchProps}) => {
  return (newBug) => {
    return fetch(url, {
      method: 'post',
      headers: {
        'Accept': 'application/json',
        'Content-Type': 'application/json'
      },
      body: JSON.stringify(newBug),
      ...fetchProps
    }).then(function (response) {
      if (!response.ok) {
        throw Error(response.statusText)
      }
      return response.json()
    })
  }
}
```

That way you can overwrite body, method, headers, or any other option on the fetch call. If this works for you I can add it probably this weekend, but until then you can use the above code snippet in your codebase if it works for you.

**Comment by ronaksvyas** [comment]
@dtschust Thanks for the reply! 
This works for me!

**Comment by dtschust** [comment]
Fixed in version 1.2.0, just published to npm!
